layouts/section/klc-history.html

```diff
@@ -0,0 +1,29 @@
+{{ partial "header.html" . }}
+
+{{ .Scratch.Set "sidemenu" true }}
+
+<div class="container">
+	<div class="row">
+		<div class="col-lg-8 col-lg-offset-2 {{ if .Scratch.Get "sidemenu" }}col-md-10 col-sm-9{{ else }}col-sm-12{{ end }}">
+			{{ partial "breadcrumb.html" . }}
+
+			<h2>KLC Revision History</h2>
```

Comment:
I'll just say I'm not a fan of version numbers in a git repo thats immediately published on every commit. This means you have to bump the version manually every time you change something in the KLC if you want any sanity. And how far do you go, fix one typo, go and bump the version?

Reply:
A new version number is only introduced if the meaning of the rules does change or if new rules are added. (So a typo does not qualify as that.)

Reply:
This should be noted somewhere as part of the PR. CONTRIBUTING.adoc is one good file to update.

Reply:
I would say details about how the version number is derived might be better at the history page it self. (Then the users have every part of this information on one page.)
But you are right a note in the contribution page on how the KLC is managed might be a good idea.